### ISSUE TITLE
add documentation for compass integration

### ISSUE THREAD

**Issue by trvsdnn** [trvsdnn/frank#18]
Need it somewhere. Maybe start working on the wiki?

**Comment by charlesroper** [comment]
Yes please!

**Comment by edrex** [comment]
I'm laying out my tools for a string of static projects, and I'm down to nanoc+Compass or Frank+Compass, but leaning towards Frank. I will put whatever I learn into the wiki. A tutorial on creating a compass+frank project would be helpful (especially for someone like me who is new to both tools).

Is [Compass-Integration](https://github.com/blahed/frank/wiki/Compass-Integration) correct? According to railsjedi's [last comment](https://github.com/blahed/frank/issues/7#issuecomment-334351) no configuration is needed now.

**Comment by trvsdnn** [comment]
It should be fine, assuming `dynamic/stylesheets` is where your stylesheets are. If you are looking at nanoc, and you find it is handled better there. I wouldn't mind hearing suggestions about how to better handle the integration.

If i remember correctly the main problem was getting plugins to load.

**Comment by wyattdanger** [comment]
Adding `@import "compass/css3"` at the top of my sass file worked fine for me the other day.

**Comment by trvsdnn** [comment]
Yeah, seems like the built in modules work okay. Using plugins like 960 https://github.com/chriseppstein/compass-960-plugin are a little different i think. You have to require the plugin, so it needs to be in scope for compass... Any luck with that?

**Comment by edrex** [comment]
The line that causes the trouble is [base.rb:250](https://github.com/blahed/frank/blob/669ef7bf7c3608b8c659c7210bf65b1b354be801/lib/frank/base.rb#L250). Copying that line into setup.rb after importing the compass plugin (or sourcing a compass config file) allows the plugin to work:

``` ruby
require 'ninesixty'
# or
Compass.add_project_configuration './compass_config.rb'

# needed to update the include path for sass etc
Frank.sass_options = Compass.sass_engine_options
```

Moving that line outside the begin..rescue..end for the compass import would allow customizing compass in setup.rb without the extra line. Or I guess another option would be to check for a hard-coded compass config file like _PROJECT_BASE/compass_config.rb_. I could make a commit for either.

**Comment by edrex** [comment]
updated https://github.com/blahed/frank/wiki/Compass-Integration
